src/Magnum/DartIntegration/Test/CMakeLists.txt

```diff
@@ -42,9 +43,9 @@ endif()
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/configure.h.cmake
                ${CMAKE_CURRENT_BINARY_DIR}/configure.h)
 
-corrade_add_test(DartIntegrationDartSkeletonTest DartSkeletonTest.cpp LIBRARIES MagnumDartIntegration ${DART_LIBRARIES})
-target_include_directories(DartIntegrationDartSkeletonTest PUBLIC ${DART_INCLUDE_DIRS})
+corrade_add_test(DartIntegrationDartWorldTest DartWorldTest.cpp LIBRARIES Magnum::OpenGLTester MagnumDartIntegration ${DART_LIBRARIES})
+target_include_directories(DartIntegrationDartWorldTest PUBLIC ${DART_INCLUDE_DIRS})
```

Comment:
Now, thinking about this, isn't adding DART include dirs done implicitly through the transitive dependency on DART?

Reply:
> Now, thinking about this, isn't adding DART include dirs done implicitly through the transitive dependency on DART?

Good catch! Thanks!